spec/features/idv/usps_verification_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'USPS verification' do
+  include SamlAuthHelper
+  include IdvHelper
+
+  context 'signing in when profile is pending USPS verification' do
```

Comment:
I thought we were planning on having these live under the `account creation` feature? I guess this is kind of a gray area since the account is already created, but it seems like it is still a part of that flow?